customers/models.py

```diff
@@ -1,17 +1,15 @@
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
-from datetime import datetime
 from app import db
+from core.mixins import BaseModelMixen
 
 # Creating the customer model
-class Customer(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
+class Customer(BaseModelMixen, db.Model):
```

Comment:
revisit how this inheritance is supposed to work. `Customer` should ideally only inherit from the mixin.